webots_ros2_urdf_converter/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
```

Comment:
I believe it is better to use the on-the-fly conversion in the `webots_ros2_universal_robot` package because we already need URDF there for MoveIt2. See:
https://github.com/cyberbotics/webots_ros2/blob/master/webots_ros2_universal_robot/resource/moveit_ur5e_description.urdf

Reply:
Good idea!
I believed there were only Webots configuration URDF file in this example as I had not noticed the `moveit_ur5e_description.urdf` file.